pkg/sql/model.go

```diff
@@ -183,3 +198,86 @@ func readGob(filePath string, object interface{}) error {
 	}
 	return nil
 }
+
+// createModelZooTable create the table "sqlflow.trained_models" to save model
+// metas the saved model URI.
+func createModelZooTable(db *DB) error {
+	createDBSQL := fmt.Sprintf("CREATE DATABASE IF NOT EXISTS %s;", modelZooDB)
+	_, err := db.Exec(createDBSQL)
+	if err != nil {
+		return err
+	}
+	// schema design: https://github.com/sql-machine-learning/sqlflow/blob/a98218ef8bee57e2a45357d7ee5721e1c6dfeb35/doc/design/model_zoo.md#model-zoo-data-schema
+	// NOTE(typhoonzero): submitter program size may exceed TEXT size(64KB)
+	// NOTE(typhoonzero): train_ir_pb contains all information how the submitter program is generated, so not saving submitter program now
+	// NOTE(typhoonzero): model_uri can be:
+	//    1. file:///path/to/model/dir
```

Comment:
`file://` or `file:///`?

Reply:
The prefix is `file://` and the path is `/path/to/model/dir` started by `/`.